src/components/PasswordReset/index.jsx

```diff
@@ -0,0 +1,114 @@
+import { Alert, Input } from "antd";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import "./index.css";
+
+const Login = (props) => {
+  const [values, setValues] = useState({ email: "" });
```

Comment:
formValues, setFormValues

Reply:
changed to 
  const [passwordResetInfo, setPasswordRestInfo] = useState({ email: "" });
based on a teammate review and will be live in the next PR